app/controllers/users_controller.rb

```diff
@@ -198,7 +198,6 @@ def login_user(user)
     store_default_taxonomy(user, 'location') unless session.has_key?(:location_id)
     TopbarSweeper.expire_cache
     telemetry_increment_counter(:successful_ui_logins)
-    user.post_successful_login
```

Comment:
This actually works for SSO logins as well see `UsersController#extlogin`, but those usually do not call `try_to_login`, thus I believe we should probably add it explicitly to the API controller (as it is going to be easier).